src/routes.ts

```diff
@@ -1,23 +1,25 @@
-import { Platform, Image, StyleProp, TextStyle } from 'react-native'
+import { Image, Platform, StyleProp, TextStyle } from 'react-native'
 import { createElement } from 'react'
 
 import {
-  createStackNavigator,
+  createAppContainer,
```

Comment:
this is imported twice and prevents transpiling

Reply:
so is `createSwitchNavigator` 😆

Reply:
probably because of some merge conflicts, will have a look on that :+1: